opentelemetry-contrib-instrumentations/src/opentelemetry/contrib-instrumentations/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.28b1"
+__version__ = "0.30b0"
```

Comment:
Did we miss this in `0.29b0`?

Reply:
I believe it was set to `0.28b1` when the PR was created but not merged until `0.29b0` was released already. So this would be its first release. (pypi does not return anything for `opentelemetry-contrib-instrumentations`)